tests/conftest.py

```diff
@@ -262,3 +266,23 @@ def some_method(self, test):
             }
         )
     build_file_tree(file_tree)
+
+
+def _containers_running(exec_path):
+    with local.cwd(exec_path):
+        if len(docker_compose("ps").splitlines()) > 2:
+            _logger.error(docker_compose("ps"))
```

Comment:
I think `docker-compose ps -q` is better for this use case.

Reply:
Thanks for the suggestion, change made.
I assume you refer to the first command, right? For the log, it makes sense to keep the full `docker-compose ps` output IMHO, to help identify the container that might be causing issues.

Reply:
Yes, sorry.